localization_manager/config/parameters.yaml

```diff
@@ -24,9 +24,15 @@ auto_initialization_timeout: 30000
 # Units: ms
 gnss_only_operation_timeout: 20000
 
+# Integer: Maximum allowed number of sequential timesteps to let lidar initialize before switching to GPS only mode
+# NOTE: Only used in GNSS only with NDT initialization mode
+# Units: Int
+sequential_timesteps_until_gps_operation: 100
```

Comment:
100 is probably a bit high. I would suggest a lower value like 5 or 10. The system rarely stays operational for that long.